README.md

```diff
@@ -52,15 +52,24 @@ Features:
 - [x] Page [middleware](https://nuxtjs.org/docs/2.x/directory-structure/middleware/)
 - [x] Basic [jsx](https://vuejs.org/v2/guide/render-function.html#JSX) (pass `h` as first argument)
 
-Modules:
+Official Modules:
 
 - [X] [http](https://http.nuxtjs.org/)
 - [x] [axios](https://axios.nuxtjs.org/)
 - [X] [i18n](https://i18n.nuxtjs.org/)
 - [x] [content](https://content.nuxtjs.org/)
+- [ ] [auth-next](https://auth.nuxtjs.org/)
```

Comment:
Module name is `auth`. Before adding to list, can you please help preparing a small repo for using vite and auth to investigate problems? (we can use a github issue)